src/components/DatePicker/DatePicker.vue

```diff
@@ -29,11 +29,11 @@
                       filled
         />
       </template>
-      <v-date-picker width="490" v-model="dateText" :min="minDate" :max="maxDate">
+      <v-date-picker id="date-picker-calendar" width="490" v-model="dateText" :min="minDate" :max="maxDate">
         <template v-slot:default>
           <div>
-            <v-btn text color="primary" @click="emitDate(dateText)"><strong>OK</strong></v-btn>
-            <v-btn text color="primary" @click="emitCancel()">Cancel</v-btn>
+            <v-btn id="btn-done" text color="primary" @click="emitDate(dateText)"><strong>OK</strong></v-btn>
+            <v-btn id="btn-cancel" text color="primary" @click="emitCancel()">Cancel</v-btn>
```

Comment:
Is there a chance of getting duplicate IDs on a page? I assume it's not possible to have multiple date pickers open at once, but are these elements rendered even if the dp isn't showing?

Reply:
I don't believe they are. In the tests i check the existence of the calendar aspect of the component before activated and it comes back as `false`, so it isn't question of displayed vs hidden, more like a `v-if` i think. 

<img width="618" alt="Screen Shot 2021-03-31 at 10 56 29 AM" src="https://user-images.githubusercontent.com/53542131/113189315-eb03ed80-920f-11eb-9bb9-74099dd7cc42.png">